7.0.0alpha2/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM debian:jessie
+
+# persistent / runtime deps
+RUN apt-get update && apt-get install -y ca-certificates curl libpcre3 librecode0 libsqlite3-0 libxml2 --no-install-recommends && rm -r /var/lib/apt/lists/*
+
+# phpize deps
+RUN apt-get update && apt-get install -y autoconf file g++ gcc libc-dev make pkg-config re2c --no-install-recommends && rm -r /var/lib/apt/lists/*
+
+ENV PHP_INI_DIR /usr/local/etc/php
+RUN mkdir -p $PHP_INI_DIR/conf.d
+
+##<autogenerated>##
+##</autogenerated>##
+
+ENV GPG_KEYS 1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763
+RUN set -xe \
+	&& for key in $GPG_KEYS; do \
+		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+	done
```

Comment:
Since we're going to have to resync with the 5.6 `Dockerfile` after the official release anyhow, I think we should leave this section out for now (since we have no `--verify` ability yet).